english/eating-walls/find-a-toilet/door/run-in/run-into-room.md

```diff
@@ -2,10 +2,12 @@ You run into the room and slam the door behind you! You don't know whose hand th
 
 The room you're now in is cold and damp, about 10x10x10ft, and made of stone.  So much for marshmallows, huh?
 
-There is an opening in the floor with a ladder going down and a lever on the wall to the right of the door.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.